### DISCUSSION TITLE
i need help

### DISCUSSION THREAD

**Discussion by Devagopal42** [ocademy-ai/machine-learning#89] [Q&A] [answered]
ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 234 is different from 236)

How do i fix this?

**Comment by ashwinn-v** [comment, marked as answer]
The number of features has to be the same for the data you trained your model and the test data you want to predict. In your case, one has 234 and the other 236. Check if you are deleting some columns by mistake or forgot to you use/delete them.

**Comment by zhangqi444** [comment]
> ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 234 is different from 236)
> 
> 
> 
> How do i fix this?

@Devagopal42  could you help share where you see this error?
